pkg/localcontroller/db/db.go

```diff
@@ -77,6 +77,19 @@ func SaveResource(name string, typeMeta, objectMeta, spec interface{}) error {
 	return nil
 }
 
+// GetResource gets resource info in db
+func GetResource(name string) *Resource {
+	r := Resource{}
+
+	queryResult := dbClient.Where("name = ?", name).First(&r)
+	if queryResult.RowsAffected == 0 {
+		klog.Warningf("resource(name=%s) not in db", name)
```

Comment:
1. leave the log to the callers, because some callers may expect non-existence
2. no Error return?

Reply:
fix it